components/profile-picture/ProfilePicture.tsx

```diff
@@ -0,0 +1,102 @@
+import { StatusBar } from "expo-status-bar";
+import React, { useEffect, useState } from "react";
+import { StyleSheet, View, Image, Pressable, Animated } from "react-native";
+import { LinearGradient } from "expo-linear-gradient";
+import * as Animatable from "react-native-animatable";
+
+type ProfilePictureProps = {
+  imageSize: number;
+  borderColor: string[];
```

Comment:
```suggestion
  borderGradientColors: string[];
```

Reply:
Depending on the status the border color will change

Reply:
You are expecting an array of strings. Prop name should be at least in plural form.
Also, you are feeding it to a LinearGradient, which means at least two colors must be provided. In this case a hint of its use in prop name seems interesting IMO